.npmignore

```diff
@@ -1,4 +1,4 @@
-/assets
+.github/
```

Comment:
WDYT of: https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d?

I think we can use `files` instead of a `.npmignore` file.